src/App.js

```diff
@@ -38,11 +36,11 @@ class App extends Component {
         </div>
         <div className="app__list">
           <List
-            tab={tasks}
+            tab={this.props.tasks}
```

Comment:
there is already state desctructuring, it would be more readable, cleaner and prettier if props will be destructured as well.